inst/include/population_dynamics/population/population.hpp

```diff
@@ -173,6 +173,7 @@ struct Population : public fims_model_object::FIMSObject<Type> {
 
     // Transformation Section
     for (size_t age = 0; age < this->nages; age++) {
+      this->weight_at_age[age] = growth->evaluate(ages[age]);
```

Comment:
question[non-blocking]: do we want to use `this->weight_at_age[ages[age]]` or `this->weight_at_age[age]` throughout the codebase?

Reply:
We discussed this when we were writing and concluded that this->weight_at_age[age] will always be the expected index value while this->weight_at_age[ages[age]] would crash if at some point we allow no integer ages. At the moment they are both equivalent though.